docs/usage/getting-started/app-secrets.md

```diff
@@ -0,0 +1,51 @@
+# Using Secrets with Mend Cloud Apps
+
+The Mend-hosted cloud apps - Renovate App on GitHub, Mend App on Bitbucket - have deprecated the use of encrypted secrets in Renovate config files stored in the repo.
+Instead, secrets for the cloud apps must be stored in the app settings using the web UI available at [developer.mend.io](http://developer.mend.io).
+
+## How to use secrets in the Mend Hosted Cloud Apps
+
+To use secrets in the Mend Cloud apps:
+
+1. Add the secret in the **Credentials** section of the [Developer Portal](http://developer.mend.io) Org or Repo settings.
```

Comment:
Need to mention here that only users with admin rights can add secrets

Reply:
Updated. Added note.